rancid-tomatillos/src/Components/Header/Header.js

```diff
@@ -1,23 +1,39 @@
 import React from 'react';
 import './Header.scss';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { removeUser, updateLoggedIn } from '../../actions/index';
 
 
-const Header = ({ data }) => {
+const Header = ({ data, isLoggedIn, removeUser, updateLoggedIn }) => {
+  let log;
+  if (isLoggedIn) {
```

Comment:
Nice work using conditional logic to determine which button is being rendered based on whether or not the user is logged in! the UX/UI is really clean for this.